config/environment.js

```diff
@@ -1,14 +1,14 @@
 // Load application config
 const logging = require('./logging');
-const secret = require('./secret');
+// const secret = require('./secret');
 const server = require('./server');
-const services = require('./services');
+// const services = require('./services');
 const swagger = require('./swagger');
 
 module.exports = {
   logging,
-  secret,
+  // secret,
   server,
-  services,
```

Comment:
Just commenting out these parts for now as they are not necessary for this PR

Reply:
Fair enough 👌 

You could take a quick look at #5 - maybe there's a quick and obvious way to get them running.